microsoft-edge/extensions/publish/add-ons-badge.md

```diff
@@ -0,0 +1,113 @@
+not in TOC
+docset: Windows > App development
+-->
+
+To promote your Microsoft Edge extension, theme, or web app, use the Microsoft Edge Add-ons badge in your websites, documentation, tutorials, and other services or integrations.  This ensures consistent, accessible, and compliant representation of the Microsoft Edge Add-ons brand.
+
+The Microsoft Edge Add-ons badge is a visual asset designed to help you promote your Microsoft Edge add-on and build user trust.
+
+The Microsoft Edge Add-ons badge reads **Get it from Microsoft Edge**:
+
+![The "Microsoft Edge Add-ons" badge](./add-ons-badge-images/get-it-from-microsoft-edge-badge.png)
+
+The Microsoft Edge Add-ons badge appears in full color with a dark background, and includes a gray border and transparent corners.  The badge `.png` files are provided in 41 languages.
+
+The Microsoft Edge Add-ons badge lets you showcase your Microsoft Edge add-on with Microsoft Edge-specific branding.  To prevent confusion with the broader [Microsoft Store](https://apps.microsoft.com), the Microsoft Edge Add-ons badge is intended exclusively for promoting add-ons that are available for Microsoft Edge.  This badge helps users clearly identify content that is built specifically for the Microsoft Edge browser.
+
+The generic Microsoft Store badge reads **Get it from Microsoft**:
+
+![The generic Microsoft Store badge](./add-ons-badge-images/generic-microsoft-store-badge.png)
+
+Unlike the generic Microsoft Store badge, the Microsoft Edge Add-ons badge provides a distinct, trusted visual cue that connects your Microsoft Edge add-on directly to the Microsoft Edge ecosystem.
+
+The Microsoft Edge Add-ons badge is not the same as the Microsoft brand logo, and they cannot be used interchangeably.
+
+
+<!-- ====================================================================== -->
+## Download the badge image files
+
+Download the complete set of `.png` files: [Microsoft Edge Add-ons Badges.zip](https://github.com/microsoft/MicrosoftEdge-Extensions/blob/main/assets/microsoft-edge-add-ons-badges.zip).
+<!-- todo:
+1. remove hidden macOS dir from the .zip file
+2. upload the Edge-specific .zip file to either:
+   download.microsoft.com (www.microsoft.com/download)
+   https://github.com/microsoft/MicrosoftEdge-Extensions/pull/365
+3. update the url above
+-->
+
+
+<!-- ====================================================================== -->
+## Badge usage guidelines
+
+Follow the required specifications for image usage, fonts, and color palette.  For detailed usage examples and guidelines, download and read the [Microsoft Edge Badge Guidelines](https://download.microsoft.com/download/0/7/D/07DF43D4-B1A8-4D38-BC02-4903BB36CEE8/Microsoft_Store_Badge_Guidelines.pdf) (PDF).
+<!-- todo:
+1. upload the Edge-specific .pdf file to either:
+   download.microsoft.com (www.microsoft.com/download)
+   https://github.com/microsoft/MicrosoftEdge-Extensions/pull/365
+2. update the url above
+-->
```

Comment:
1. upload the Edge-specific .pdf file to either:
   download.microsoft.com (www.microsoft.com/download)
   https://github.com/microsoft/MicrosoftEdge-Extensions/pull/365
2. update the url above